POI/categories.go

```diff
@@ -51,6 +56,25 @@ func GetPlaceTypes(placeCat PlaceCategory) (placeTypes []LocationType) {
 	return
 }
 
+// Returns whether the eatory place is pricy based on its price level
+func IsPricyEatery(placeCategory PlaceCategory, priceLevel PriceLevel) bool {
```

Comment:
Consider simplify to return `placeCategory == PlaceCategoryEatery && priceLevel >= PriceLevelThree`

Reply:
Done